subt/subt-ssci-x2.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "app": {
+        "init": {
+          "limit_pitch": 40,
+          "limit_roll": 35,
+          "return_limit_pitch": 45,
+          "return_limit_roll": 45,
+          "speed_policy": "conservative"
+        }
+      },
+      "depth2scan": {
+        "init": {
+          "depth_params": {
+            "fx": 554.3,
+            "camera_xyz": [0.23, 0.0, 0.19],
+            "camera_ypr": [0.0, 0.3491, 0.0],
+            "image_size": [640, 480],
+            "principal_point": [320.5, 240.5],
+            "min_x": 0.25,
+            "lidar_fov": 6.283185307179586,
```

Comment:
Note: This needs https://github.com/robotika/osgar/pull/609 to work.

Reply:
already merged, thanks for reminder m.

Reply:
For posterity sake: https://github.com/osrf/subt/blob/60aeb723be6af834c1499221f2a7ad16f926321c/submitted_models/ssci_x2_sensor_config_1/model.sdf#L235-L278
The params seem to match what is in the sdf file.